content/patterns/amd-rag-chat-qna/amd-rag-chat-qna-required-hardware.adoc

```diff
@@ -0,0 +1,41 @@
+---
+title: Required Hardware
```

Comment:
Global comment: Write all titles and headings, in sentence-style capitalization.
```suggestion
title: Required hardware
```